src/babelRegister.js

```diff
@@ -20,15 +20,17 @@ function resolve(presetsOrPlugins, type) {
  * To prevent `babel-register` from reading foreign Babel
  * configs, `babelrc` has to be set to `false` and Haul's
  * Babel config inlined into `babel-register`'s options.
- * 
+ *
  * Also presets and plugins need to be resolved to
  * Haul's `node_modules` directory, otherwise Babel would
  * search from them in project's `node_modules`.
  */
 require('babel-register')(
   Object.assign(
     {
-      ignore: /node_modules(?!\/haul)/,
+      // Replaces Haul's 'ignore' pattern, which does not work with PNPM.
+      // eslint-disable-next-line no-useless-escape
+      ignore: /node_modules(?!.*[\/\\]haul)/,
```

Comment:
Can you remove this useless escape inside character set? 
This will be enough
```
/node_modules(?!.*[/\\]haul)/
```

Reply:
d'oh. Thanks!